src/plugins/synced-lyrics/renderer/components/LyricsPicker.tsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable import/order,@typescript-eslint/no-unused-vars */
+
+import { createSignal, For } from 'solid-js';
+import type { YtIcons } from '@/types/icons';
+import { providers } from '@/plugins/synced-lyrics/providers';
+
+export const [providerIdx, setProviderIdx] = createSignal(0);
+export const LyricsPicker = () => {
+  const next = () => setProviderIdx((i) => (i + 1) % providers.length);
+  const previous = () => setProviderIdx((i) => (i - 1 + providers.length) % providers.length);
+
+  const chevronLeft: YtIcons = 'yt-icons:chevron_left';
+  const chevronRight: YtIcons = 'yt-icons:chevron_right';
```

Comment:
as seen here, I have saved all the icon names as string unions, so it is easy to discover/use an icon